code/datums/helper_datums/teleport.dm

```diff
@@ -188,7 +188,7 @@
 	//VOREStation Edit Start
 	if(destination.z in using_map.player_levels)
 		return 1
-	if(istype(teleatom, /mob/living))
-		to_chat(teleatom, "<span class='warning'>The portal refuses to carry you that far away!</span>")
+	//if(istype(teleatom, /mob/living))

```

Comment:
Do a check to see if their belly has over 3 mobs.

If there's over 3 mobs then refuse to teleport.

There's so many things I dont recommend with televore it's so fustrating to balance. It's why they disabled it originally.

Reply:
Will do. Do you mean mobs, or just items?